README.md

```diff
@@ -50,6 +50,7 @@ This is a collection of awesome resources related to the yearly
     * [PowerShell](#powershell)
     * [Prolog](#prolog)
     * [Python](#python)
+      * [smetanin-av/advent_of_code](https://github.com/smetanin-av/advent_of_code.git)
```

Comment:
This isn't the right place to put your entry. Search for the `## Python` section below and add it there with the rest of the entries.